extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -363,10 +392,6 @@ public function getTitle() {
 	public function getData( $master = false ) {
 		global $wgUser, $wgBlankImgUrl, $wgMemc, $wgArticleCommentsEnableVoting;
 
-		wfProfileIn( __METHOD__ );
-
-		$comment = false;
```

Comment:
The `$comment` variable is set to false and used as a return value for failure until its overwritten with an actual comment below.  Updated to return false directly in those places and not bothering to define it up here just to hold a non-comment boolean.